jscomp/test/dist/jscomp/test/js_int_test.js

```diff
@@ -9,7 +9,7 @@ var suites_0 = [
       return {
               TAG: /* Eq */0,
               _0: "1.23456e+5",
-              _1: (123456).toExponential()
+              _1: (123456).toExponential(undefined)
```

Comment:
Interesting. This only happens when no labeled args are passed, right? Hopefully it doesn't have any impact on other JS apis 🤔

Reply:
yeah. I checked every function to make sure the `undefined` argument is not an issue.

In the case of `Js.Date`, for example, we need to keep separate functions, as the `undefined` argument can make the functions return `NaN`.